eng/ci/code-mirror.yml

```diff
@@ -12,7 +12,7 @@ resources:
       ref: refs/tags/release
 
 variables:
-  - template: ci/variables/cfs.yml@eng
+  - template: /ci/variables/cfs.yml@eng
 
 extends:
-  template: ci/code-mirror.yml@eng
\ No newline at end of file
+  template: /ci/code-mirror.yml@eng
```

Comment:
this should be /eng/ci

Reply:
I'm not sure there is an eng directory in this repo. All the documentation references I've seen refer to it as `ci/<name>.yml@eng`